build/flatpak/metainfo.xml.in

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>${FLATPAK_ID}</id>
+  <name>${FLATPAK_APPNAME}</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>New Vector Ltd</developer_name>
+  <summary>Create, share, communicate, chat and call securely, and bridge to other apps</summary>
```

Comment:
We can probably find some copy for this: I would guess the "Secure and independent communication, connected via Matrix" from the homepage.